src/plugins/controls/public/time_slider/components/time_slider.tsx

```diff
@@ -78,7 +78,7 @@ export const TimeSlider: FC<Props> = (props: Props) => {
         value={[from, to]}
         onChange={props.onChange}
         onClear={() => {
```

Comment:
`TimeSliderPopoverContent.onClear` prop is no longer needed and should not be passed to TimeSliderPopoverContent

Reply:
Oooh good catch - fixed in 3893cf5ad20370b448410cf8f1420e4c014d32cb :+1: